cpp/include/cugraph/detail/shuffle_wrappers.hpp

```diff
@@ -93,7 +94,8 @@ std::tuple<rmm::device_uvector<vertex_t>,
            rmm::device_uvector<vertex_t>,
            std::optional<rmm::device_uvector<weight_t>>,
            std::optional<rmm::device_uvector<edge_t>>,
-           std::optional<rmm::device_uvector<edge_type_id_t>>>
+           std::optional<rmm::device_uvector<edge_type_id_t>>,
+           std::vector<size_t>>
```

Comment:
New final parameter is not described in the documentation above.

Reply:
Right. We are returning the `rx_count`. I am adding that to the documentation in my next commit